Gemfile

```
@@ -0,0 +1,49 @@
+source 'https://rubygems.org'
```

thay thế dùng nháy kép trong file này em nhé
vd: 
```suggestion
source "https://rubygems.org"
```